lib/pa_ess/utilities.ex

```diff
@@ -579,7 +580,8 @@ defmodule PaEss.Utilities do
     {"Gallivan Blvd", "881"},
     {"Brookline Ave", "885"},
     {"Brookline Village", "886"},
-    {"Cobbs Corner Canton", "887"}
+    {"Cobbs Corner Canton", "887"},
+    {"Linden Sq", "889"}
```

Comment:
Should this be the original headsign rather than the shortened one, i.e. "Linden Square"? The one above, "Cobbs Corner Canton", uses the full headsign.

Reply:
The caller that references this list uses `String.starts_with?/2` https://github.com/mbta/realtime_signs/blob/main/lib/pa_ess/utilities.ex#L678 so we don't _need_ to use the original headsign. I decided to use `Linden Sq` to be a little more flexible in case the headsign ever happens to change for some reason and come in as the abbreviated version.